src/ofdm_internal.h

```diff
@@ -88,7 +88,7 @@ struct OFDM_CONFIG {
     int ns;  /* Number of Symbol frames */
     int bps;   /* Bits per Symbol */
     int txtbits; /* number of auxiliary data bits */
-    int phase_est_bandwidth; /* force low of high phase est bandwidth, rather than letting state machine decide */
+    int phase_est_bandwidth; /* select low, high, or locked high phase est bandwidth, rather than letting state machine decide */
```

Comment:
So maybe add a ```phase_est_bandwidth_mode``` variable here that controls how phase_est_bandwidth is switched.

Reply:
Very good, will make another pass with those changes.